README.md

```diff
@@ -1 +1,68 @@
 # The-E-Commerce-Backend
+
+A Backend for an E-Commerce system that has been written in NodeJS with the aid of ExpressJS as a server/router, sequelize as an ORM for MySQL database. The operation was tested with Insomina.
+
+![Product screenshot](./screenshots/screenshot1.png)
+
+## Table of Contents
+
+* [Technologies](#technologies)
+* [Installation](#installation)
+* [Feature](#feature)
+* [Usage](#usage)
+* [License](#license)
+* [Questions](#questions)
+* [Screens](#screens)
+
+
+## Technologies 
+
+* JavaScript ES6
+* Node.js
+* npm
+* MySQL
+* mysql2
+* Sequelize ORM
+* Insomnia
+* Nodemon
+
+## Installation 
+
+Following are the steps of installation
+
+```
+npm i
```

Comment:
might be nice to add some guidance about setting uo the db and seeding it here :-)